pyccel/ast/numpytypes.py

```diff
@@ -118,7 +118,6 @@ class NumpyInt8Type(NumpyIntType):
     _name = 'numpy.int8'
     _precision = 1
 
-
 class NumpyInt16Type(NumpyIntType):
```

Comment:
Bad change. This is unnecessary for this PR and breaks PEP recommendations : https://peps.python.org/pep-0008/#blank-lines

> Surround top-level function and class definitions with two blank lines.

Reply:
fixed